### DISCUSSION TITLE
[Windows: 版本3.1.1.0/3.1.3/0.51.0-sakura-7] 未知错误，tls 握手失败

### DISCUSSION THREAD

**Discussion by ziangchen7** [natfrp/wiki#512] [使用问题] [answered]
2024/07/12 01:05:13 E Service 登录失败: 未知错误 (Get "https://natfrp-api.globalslb.net/v4/user/info": tls: first record does not look like a TLS handshake)

2024/07/12 01:28:33 E Service/Updater 更新检查失败: Post "https://natfrp-api.globalslb.net/v4/system/update": tls: first record does not look like a TLS handshake

我的设备上sakura frp突然无法登录和更新，原因还不知道，从某个时间段之后就这样了
没有代理之类的打开
无法登录所以进入不了隧道。
其他设备上没有这样的问题。
有尝试卸载重复安装。

![image](https://github.com/user-attachments/assets/4bae04b7-cf47-4438-8996-967e5668df32)

![image](https://github.com/user-attachments/assets/9d24fc75-0c18-422c-b961-adaaed8acdd4)

非常感谢！

[a comment by an automated account is not shown]

**Reply by ziangchen7** [reply]
问题打开代理后以解决，非常感谢！

**Comment by DylanLee-152406** [comment]
我也遇到了这个问题, 我看了回答没看懂，请问具体咋解决，我的电脑是macbook
